rpcs3/Emu/RSX/RSXFIFO.cpp

```diff
@@ -30,9 +30,17 @@ namespace rsx
 			}
 		}
 
+		template <int full>
```

Comment:
Just use bool here. read<0> is unintuitive and makes it seem like you're accessing element 0 of something.